pandas/core/config_init.py

```diff
@@ -466,3 +466,14 @@ def _register_xlsx(engine, other):
     except ImportError:
         # fallback
         _register_xlsx('openpyxl', 'xlsxwriter')
+
+# Set up the io.parquet specific configuration.
+parquet_engine_doc = """
+: string
+    The default parquet reader/writer engine. Available options:
+    None, 'pyarrow', 'fastparquet'
+"""
```

Comment:
What is the default?

Also, you can you add this option to the options.rst docs